canopen/node/local.py

```diff
@@ -39,6 +39,9 @@ def __init__(
         self.emcy = EmcyProducer(0x80 + self.id)
 
     def associate_network(self, network: canopen.network.Network):
+        if self.network is not canopen.network._UNINITIALIZED_NETWORK:
+            # Unsubscribe from old network (to prevent double subscription)
+            self.remove_network()
```

Comment:
Is it really common to re-associate a node to a different network?  I would think raising a `RuntimeError` exception here is more appropriate.